src/components/video-card/VideoCard.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { MoreOptionsIcon } from "../../icons/Icons";
+export function VideoCard({ item }) {
+  return (
+    <div className="card card-vrt">
+      <div className="card-img-container">
+        <img className="card-img" src={item.thumbnail} alt={item.alt} />
+      </div>
+      <div className="card-desc">
+        <div className="avatar avatar-ss">
+          <img src={item.avatar} alt="avatar" className="circle-img" />
+        </div>
+        <div className="flex-1 m-l-1">
+          <p>{item.title.length > 50 ? item.title.substring(0,51)+'...' : item.title}</p>
```

Comment:
Suggestion: The idea is nice to use `...` here, check if you can add styling in CSS to use ellipses rather than handling the length display here.

Reply:
sure, will search and implement